### ISSUE TITLE
Won't load if JSONView extension is also enabled

### ISSUE THREAD

**Issue by JustinFarris** [flightstats/hubNavigator#2]

**Comment by linusred** [comment]
Yep! It's known. I'd really like it to play nicely with other formatters. I'm working on it. It's a tough problem, because none of the current chrome extensions implement cross extension message passing, or parse things through AJAX calls. I'm looking into it though.
